@xen-orchestra/proxy/src/main/mixins/api.js

```diff
@@ -81,6 +99,43 @@ export default class Api {
     httpServer.on('request', koa.callback())
 
     this.addMethods({
+      system: {
+        getMethodsInfo: [
+          getMethodsInfo.bind(this),
```

Comment:
Why don't inline them like `methodSignature`?

Reply:
No major reasons, I will fix this.